site/_includes/macros/cards/blog-card.njk

```diff
@@ -3,12 +3,9 @@
 {% macro blogCard(post) %}
 {% set pd = post.data %}
 <div class="blog-card rounded-lg pad-300 md:pad-400 width-full">
-  {% if pd.thumbnail %}
-  {% set thumbnail = pd.thumbnail %}
-  {% else %}
-  {% set thumbnail = pd.hero %}
-  {% endif %}
-  {% set alt = pd.alt %}
+  {% set fallbackImg = 'image/SZHNhsfjU9RbCestTGZU6N7JEWs1/Wa6Eb8lgDc7Q4ugFV2bw.svg' %}
```

Comment:
Hi @mamieorine , let's not hardcode strings in random njk templates - very hard to find them after a while. Please add this url somewhere next to https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_data/site.json#L21

Reply:
I have moved the hardcode image URL to /site.json